File: JavaScript/calc.js
const prompt = require('prompt-sync')();

console.log("=== Calculator ===");
console.log("1 = +");
console.log("2 = -");
console.log("3 = x");
console.log("4 = /");
const input = prompt("Enter number : ")

function operation() {
    const num1 = parseInt(prompt("First Number : "));
    const num2 = parseInt(prompt("Second Number : "));
    if (input == 1){
        return(console.log(num1+num2));
    } else if (input == 2){
        return(console.log(num1-num2));
    } else if (input == 3){
        return(console.log(num1*num2));
    } else if (input == 4){
        return(console.log(num1/num2));
    }
}
if (input == 1){
    operation()
} else if (input == 2){
    operation()
} else if (input == 3){
    operation()
} else if (input == 4){
    operation()
} else {
    console.log("Invalid Input")
}

File: Python/calc.py
print("=== Calculator ===")
print("1 = +")
print("2 = -")
print("3 = x")
print("4 = /")
operation = int(input("Enter Operation : "))

def calc():
    num1 = int(input("First Number : "))
    num2 = int(input("Second Number : "))
    if operation == 1 :
        print(num1 + num2)
    if operation == 2 :
        print(num1 - num2)
    if operation == 3 :
        print(num1 * num2)
    if operation == 4 :
        print(num1 / num2)

if operation == 1 :
    calc()
elif operation == 2 :
    calc()
elif operation == 3 :
    calc()
elif operation == 4 :
    calc()
else :
    print("Invalid input")
